tests/unittests/EntryPoint.cpp

```diff
@@ -17,19 +17,23 @@
 #include <windows.h>
 #include <TestFramework.h>
 
+#ifdef WIN32
+#include <wrl\wrappers\corewrappers.h>
+using namespace Microsoft::WRL::Wrappers;
+#endif
+
 int main(int argc, char** argv) {
-// ::CoInitialize, Uninitialize, don't exist on OSX, aren't needed
 #ifdef WIN32
-    // Initialize COM for all of the tests
-    ::CoInitializeEx(NULL, COINIT_MULTITHREADED);
+    // Initialize the windows runtime, with uninitialized upon destructor invocation.
+    RoInitializeWrapper initialize(RO_INIT_MULTITHREADED);
+    if (FAILED(initialize)) {
+        return -1;
+    }
 
     testing::InitGoogleTest(&argc, argv);
     LOG_INFO("Starting unit tests...");
```

Comment:
>LOG_INFO [](start = 4, length = 8)

nit: if the message doesn't have to come after the InitGoogle call you could move the init and result lines out of the ifdef blocks.